.ci/azure-pipelines.yml

```diff
@@ -17,6 +17,12 @@ jobs:
       submodules: true
       persistCredentials: true
 
+    - task: CmdLine@2
+      displayName: "Markdownlint"
+      inputs:
+        script: 'npm install -g markdownlint-cli && markdownlint **/*.md --ignore node_modules --ignore src'
+        workingDirectory: '$(Build.SourcesDirectory)'
```

Comment:
I would separate the install and the run into two tasks, and I also think our other pipelines use npx rather than install the libraries globally.